RELEASE_NOTES

```diff
@@ -1,5 +1,8 @@
 $Id$
 
+3.42 2024/06/04 (cvuosalo)
+    - Add support for https in URLs specified in servlets.conf.
```

Comment:
Change `servlets.conf` to `FileBaseDirectory`.  The former is a frontier-tomcat rpm concept, not a frontier servlet concept.